tech/languages/java/java-build-tools-installation.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Java Build Tools
+page: java
+---
+
+## Maven installation
+
+Maven is probably the most popular build tool in Java world these days. It's available in official Fedora repositories, in form of RPM. To install it, simply type:
+
+```
+# dnf install maven
+```
```

Comment:
Long story short, Java is special :) Using artifacts from RPMs for development is hard and I fully understand why people don't use them. Projects like [Language Specific Repositories] could improve the situation significantly, but it's a far cry.

[Language Specific Repositories]: https://fedoraproject.org/wiki/Env_and_Stacks/Projects/LanguageSpecificRepositories